src/main/java/webs/hillel/StudentDAO.java

```
@@ -0,0 +1,78 @@
+package webs.hillel;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class StudentDAO {
+    private static final Logger logger = Logger.getLogger(StudentDAO.class.getName());
+
+    private final SessionFactory sessionFactory;
+
+    public StudentDAO() {
+        sessionFactory = HibernateSession.getSessionFactory();
+    }
+
+    public void addStudent(Student student) {
+        Transaction transaction = null;
+        try (Session session = sessionFactory.openSession()) {
+            transaction = session.beginTransaction();
+            session.save(student);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            logger.log(Level.SEVERE, "Failed to add student: " + e.getMessage(), e);
```

Instead of logging would be better to throw an exception here